src/fe/inf_fe_map.C

```diff
@@ -187,6 +187,13 @@ Point InfFE<Dim,T_radial,T_map>::inverse_map (const Elem * inf_elem,
            +o(2)*p1(0)*p0(1)+o(0)*p0(2)*p1(1));
 
 
+        if(! std::isfinite(c_factor)){
```

Comment:
We use C99 for isnan() if the user has a C99 C compiler, but even in that case we fall back on the IEEE "a != a" isnan-equivalent if the user doesn't have a C99 C compiler.

We definitely *shouldn't* be requiring a C99-cum-C++98 C++ compiler.  Such frankencompilers did exist, but they're technically non-conforming to *either* standard.  IIRC we added the libmesh_isnan.c circumlocutions after a compiler upgrade took *away* C99 isnan().

The right thing to do here is probably another shim function, but without anything so ridiculous as a C file this time.  libmesh_isfinite() could use std::isfinite() if C++11 is supported, or test ```a > numeric_limits<T>::max() || a < -numeric_limits<T>::max()``` otherwise?